ddtrace/appsec/_iast/_taint_tracking/aspects/aspect_index.cpp

```diff
@@ -81,7 +81,7 @@ api_index_aspect(PyObject* self, PyObject* const* args, const Py_ssize_t nargs)
     }
 
     TRY_CATCH_ASPECT("index_aspect", return result_o, , {
-        const auto tx_map = Initializer::get_tainting_map();
+        const auto tx_map = taint_engine_context->get_tainted_object_map(candidate_text);
```

Comment:
nit (and maybe outside of this PR context since I don't think `get_tainted_object_map` is defined here): since only objects can be tainted, `get_tainted_map` would be shorter and enough.

Reply:
In previous versions, it could be understood that `get_tainting_map` returned a map of ranges; with this name, it's now clearer that it returns a map of `TaintedObject`